src/lib/operators/abstract_operator.cpp

```diff
@@ -168,14 +164,16 @@ void AbstractOperator::execute() {
 }
 
 std::shared_ptr<const Table> AbstractOperator::get_output() const {
-  Assert(_executed, "Trying to get_output of operator that was not executed yet.");
+  Assert(executed(), "Trying to get_output of operator that was not executed yet.");
```

Comment:
Trying to remember - did we already discuss if `get_output` on a cleared operator is valid? If not, we should assert that we are exactly in the executed state (see the ambiguity I commented on earlier). If, for some reason, we need this to be valid, this should be doc'ed.

Reply:
I think we did. 
However, there are many tests that do `EXPECT_EQ(op->get_output(), nullptr)`

Reply:
> I think we did.

Do you remember what the result was? Unfortunately, I can't. Looking at this again, I can't come up with a good reason why anyone should call `get_output` on a cleared operator.

>  However, there are many tests that do EXPECT_EQ(op->get_output(), nullptr)

When searching for `get_output(), nullptr)`, I only found `operator_clear_output_test.cpp`. Obviously, that would need to be adapted and changed to `EXPECT_THROW`.